pkgs/development/tools/build-managers/gn/default.nix

```
@@ -2,9 +2,9 @@
 
 callPackage ./generic.nix args {
   # Note: Please use the recommended version for Chromium, e.g.:
-  # https://git.archlinux.org/svntogit/packages.git/tree/trunk/chromium-gn-version.sh?h=packages/gn
-  rev = "fd3d768bcfd44a8d9639fe278581bd9851d0ce3a";
-  revNum = "1718"; # git describe HEAD --match initial-commit | cut -d- -f3
-  version = "2020-03-09";
-  sha256 = "1asc14y8by7qcn10vbk467hvx93s30pif8r0brissl0sihsaqazr";
+  # https://github.com/archlinux/svntogit-packages/blob/packages/gn/trunk/chromium-gn-version.sh
```

Arch Linux recently completed their git migration: https://archlinux.org/news/git-migration-completed/

```suggestion
  # https://gitlab.archlinux.org/archlinux/packaging/packages/gn/-/blob/main/chromium-gn-version.sh
```

---

The (OmahaProxy)[https://omahaproxy.appspot.com] service which was used by that script is now permanently offline and Arch simply dropped the script for good. Instead I am referring to the chromium nix file, which contains the gn version/rev as well.